examples/SimpleRagas/SimpleRagas.ipynb

```diff
@@ -0,0 +1,1018 @@
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Experiment No Document Selection, TopK=5, Model=gpt-4o-b1d1a2d3 is running at https://www.braintrust.dev/app/braintrustdata.com/p/Ragas%20Retrieval/experiments/No%20Document%20Selection%2C%20TopK%3D5%2C%20Model%3Dgpt-4o-b1d1a2d3\n",
+      "Ragas Retrieval [experiment_name=No Document Selection, TopK=5, Model=gpt-4o] (data): 20it [00:00, 42388.12it/s]\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "b280db484ac748c8bd7caee41d614cd6",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Ragas Retrieval [experiment_name=No Document Selection, TopK=5, Model=gpt-4o] (tasks):   0%|          | 0/20 […"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "=========================SUMMARY=========================\n",
+      "No Document Selection, TopK=5, Model=gpt-4o-b1d1a2d3 compared to No Document Selection, Topk=5, model=gpt-3.5-turbo-dbcd2905:\n",
+      "51.41% (-) 'ContextEntityRecall' score\t(0 improvements, 0 regressions)\n",
+      "67.36% (+06.37%) 'AnswerCorrectness'   score\t(12 improvements, 8 regressions)\n",
+      "100.00% (-) 'Faithfulness'        score\t(0 improvements, 0 regressions)\n",
+      "23.64% (-) 'ContextRelevancy'    score\t(0 improvements, 0 regressions)\n",
+      "\n",
+      "2.48s (-216.46%) 'duration'\t(14 improvements, 6 regressions)\n",
+      "\n",
+      "See results for No Document Selection, TopK=5, Model=gpt-4o-b1d1a2d3 at https://www.braintrust.dev/app/braintrustdata.com/p/Ragas%20Retrieval/experiments/No%20Document%20Selection%2C%20TopK%3D5%2C%20Model%3Dgpt-4o-b1d1a2d3\n"
+     ]
+    }
+   ],
+   "source": [
+    "result = await Eval(\n",
+    "    name=\"Ragas Retrieval\",\n",
+    "    experiment_name=f\"No Document Selection, TopK={TOP_K_TO_USE_NO_LLM}, Model={QA_ANSWER_MODEL}\",\n",
+    "    data=[\n",
+    "        {\n",
+    "            \"input\": x[\"input\"],\n",
+    "            \"expected\": x[\"expected\"],\n",
+    "            \"metadata\": {\"context\": x[\"context\"]},\n",
+    "        }\n",
+    "        for x in rag_results_w_context_no_llm_selection\n",
+    "    ],\n",
+    "    task=lambda input: q_and_a_dict[input],\n",
+    "    scores=[\n",
+    "        answer_correctness,\n",
+    "        context_entities,\n",
+    "        context_relevancy,\n",
+    "        # context_recall,\n",
+    "        faithfulness,\n",
+    "    ],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Comparing in Braintrust\n",
+    "\n",
+    "The results here intuitively make sense:\n",
+    "\n",
+    "- When we move from using an LLM to pick the most relevant docs to simply using more docs, Context Relevancy decreases\n",
```

Comment:
i think it would be useful to explain what context relevancy is at some point (probably above).

Also maybe inject a couple opinions about how to best utilize RAGAS, like "we've found that these metrics are the best" or something.

Right now it reads a bit more like "we observe that RAGAS behaves like this" vs. "you can really get useful insights from RAGAS if you do ___"

Reply:
The coolest way to do this would be to use braintrust to manually label answers as correct/incorrect, and then describe what rag metrics correlate to them